core/src/main/scala/com/mesosphere/package.scala

```diff
@@ -0,0 +1,78 @@
+package com
+
+import com.typesafe.scalalogging.CanLog
+import com.typesafe.scalalogging.LoggerTakingImplicit
+import org.slf4j.MDC
+import scala.language.implicitConversions
+
+package object mesosphere {
+
+  type KvArgs = Traversable[(String, Any)]
+
+  /**
+   * By default, we use an empty context for all loggers. This is needed where we have a class/object
+   * level implicit logger but it may not always be needed/required to have an implicit [[KvArgs]] in scope.
+   */
+  implicit val emptyKvArgs : KvArgs = Traversable()
+
+  implicit def tupleToTraversable(kv: (String, Any)): KvArgs = Traversable(kv)
```

Comment:
why do we need this implicit? 🤔

Reply:
This `tupleToTraversable` was used implicitly a couple times in this class (See the overloaded `and` method in `KvArgsOps` class). I can remove the `implicit` however, if that's a concern.